product/views/images_view.py

```
@@ -0,0 +1,43 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework.parsers import FileUploadParser
```

Ruff (0.8.2)

`rest_framework.parsers.FileUploadParser` imported but unused

Remove unused import: `rest_framework.parsers.FileUploadParser`

(F401)